Cargo.toml

```diff
@@ -212,6 +212,7 @@ near-primitives-core = { path = "core/primitives-core" }
 near-rosetta-rpc = { path = "chain/rosetta-rpc" }
 near-rpc-error-core = { path = "tools/rpctypegen/core" }
 near-rpc-error-macro = { path = "tools/rpctypegen/macro" }
+near-rust-allocator-proxy = { path = "../near-memory-tracker/near-rust-allocator-proxy" }
```

Comment:
this local dependency needs to be replaced with one that points to a version with https://github.com/near/near-memory-tracker/pull/104 merged.